content/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md

```diff
@@ -812,8 +802,8 @@ reasons](https://github.com/kubernetes/community/pull/1982). To use
 `RotateKubeletServerCertificate` operators need to run a custom approving
 controller, or manually approve the serving certificate requests.
 -->
-Kubernetes 核心中所实现的 CSR 批复控制器出于
-[安全原因](https://github.com/kubernetes/community/pull/1982)
+Kubernetes 核心中所实现的 CSR
+批复控制器出于[安全原因](https://github.com/kubernetes/community/pull/1982)
 并不会自动批复节点的**服务**证书。
```

Comment:
```suggestion
批复控制器出于[安全原因](https://github.com/kubernetes/community/pull/1982)并不会自动批复节点的**服务**证书。
```

Reply:
(I personally think it's acceptable (but should be avoid) to break line because of long link.)

Reply:
I agree and you can change it or not.

Reply:
reorganized the sentence.